docs/Get-HelpPreview.md

```diff
@@ -34,44 +34,43 @@ Update-MarkdownHelpModule
 Update-MarkdownHelpSchema
 ```
 
-Returns a help object get-help preview from maml xml and assign it to the $help variable.
-Gets the names of Cmdlet objects inside help.
+The first command creates a **Help** object for the the specified MAML file. The command stores it in the $Help variable.
+
+The second command displays the **Name** property for each of the objects in $Help.
+
 ## PARAMETERS
 
 ### -Path
-Path to MAML help files.
-You can pass several of them.
+Specifies an array of paths of MAML external help files.
 
 ```yaml
 Type: String[]
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
-### CommonParameters
```

Comment:
I understand, but the change that I made was to make the link live. I guess I should check with Tara to see if that would be an option for the rest of the topics -